tests/paddle/test_operators.py

```diff
@@ -296,6 +294,55 @@ def test_gelu_bwd_fp8(fp8_dtype):
         assert_allclose(x_grad_t, x.grad.T, rtol=0.1, atol=0.01)
         assert_allclose(dbias, x.grad.sum(axis=0), rtol=0.1, atol=0.01)
 
+    @staticmethod
+    def test_silu_bf16():
+        """
+        Test BF16 SiLU Forward
+        """
```

Comment:
SwiGLU and SiLU/swish are different activations:
```suggestion
    def test_swiglu_bf16():
        """
        Test BF16 SwiGLU Forward
        """
```
There are a few other tests that are called "silu" but use SwiGLU.

Reply:
Done